about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Jake & Elwood</title>
+        <!-- viewport meta to render pages on mobile properly -->
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+        <!-- link to google fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700&display=swap" rel="stylesheet">
+        
+        <!-- https://cdnjs.com/libraries/font-awesome -->
+        <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.css" rel="stylesheet">
+        
+        <!-- link to my CSS -->
+        <link href="styles.css" rel="stylesheet">
+    </head>
+     <body>
+        <section class="main-wrapper">
+            <header class="header-home">
+                
+            
+                    <div class="header-content-1">
+                        <section class="nav-1-align">
+                        <div class="align-left"><p></p></div>
+                        <div class="nav-1 ">
+                            <ul class="nav-list-1">
+                                <li class="nav-item-1"><a href="#" class="nav-link-1"><i class="fab fa-twitter"></i></a></li>
+                                <li class="nav-item-1"><a href="#" class="nav-link-1"><i class="fab fa-facebook-square"></i></a></li>
+                                <li class="nav-item-1"><a href="#" class="nav-link-1">English</a></li>
+                                <li class="nav-item-1"><a href="#" class="nav-link-1">MyPage</a></li>
+                                <li class="nav-item-1"><a href="#" class="nav-link-1">Logout</a></li>
+                            </ul>
+                        </div>
+                       </section>
```

Comment:
- [Optional] Please use an indentation of two spaces in all your documents for readability as this is a convention and also [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).